src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,55 @@
+import classNames from 'classnames';
+import { FilterType } from '../../helpers/filterTodos';
+
+interface Props {
+  activeTodosCount: number;
+  hasCompletedTodo: boolean;
+
+  removeAllCompleted: () => void;
+  filterType: FilterType;
+  setFilter: React.Dispatch<React.SetStateAction<FilterType>>;
+}
+
+export const Footer: React.FC<Props> = ({
+  activeTodosCount,
+  hasCompletedTodo,
+
+  removeAllCompleted: deleteAllCompleted,
+  filterType,
+  setFilter,
+}) => {
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${activeTodosCount} items left`}
```

Comment:
Use template literals directly in the JSX markup to avoid unnecessary string interpolation. For example, `<span className="todo-count">{activeTodosCount} items left</span>`.